packages/client-utils/package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "@agoric/client-utils",
+  "version": "0.1.0",
+  "description": "Utilities for building Agoric clients",
+  "license": "Apache-2.0",
+  "type": "module",
+  "main": "src/main.js",
+  "files": [
+    "src"
+  ],
+  "scripts": {
+    "build": "exit 0",
+    "prepack": "tsc --build tsconfig.build.json",
+    "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
+    "test": "ava",
+    "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
+    "test:xs": "exit 0",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc",
+    "lint:eslint": "eslint ."
+  },
+  "devDependencies": {
+    "ava": "^5.3.0",
+    "c8": "^9.1.0",
+    "ts-blank-space": "^0.4.1"
+  },
+  "dependencies": {
+    "@agoric/cache": "^0.3.2",
+    "@agoric/casting": "^0.4.2",
```

I don't see any imports of casting.

**request**:

```suggestion
    "@agoric/casting": "^0.4.2",
```

---

there is now